search/includes/classes/class-search.php

```diff
@@ -764,7 +764,7 @@ public static function should_load_es_wp_query() {
 	/**
 	 * Filter ElasticPress index name if using VIP ES infrastructure
 	 */
-	public function filter__ep_index_name( $index_name, $blog_id, $indexable ) {
+	public function filter__ep_index_name( $_index_name, $blog_id, $indexable ) {
```

Comment:
Introduce a new variable instead of reusing the parameter "$index_name".

Because the parameter is not used, it is easier to rename the parameter than the variable.

Reply:
Noting that with PHP 8, this would technically be a breaking change, due to the Named Parameters feature.